src/moderatorFrontendMain/kotlin/app/AppContent.kt

```diff
@@ -101,12 +65,6 @@ interface AppContentClasses {
 private val AppContentStyle = { theme: dynamic ->
   // Keep in sync with AppContentClasses!
   js {
```

Comment:
container is now undefined? should we keep it in sync as written in the comment?